File: 01a.pl
#! /usr/bin/perl
use warnings;
use strict;
use feature qw{ say };

use ARGV::OrDATA;
use List::Util qw{ sum };

local $/ = "";

my $max = 0;
while (<>) {
    my $sum = sum(split /\s+/);
    $max = $sum if $sum > $max;
}

say $max;

__DATA__
1000
2000
3000

4000

5000
6000

7000
8000
9000

10000

File: 01a.py
#!/usr/bin/python3
import sys

max = 0
with open(sys.argv[1], 'r') as fh:
    sum = 0
    for line in fh:
        if len(line) > 1:
            sum += int(line.rstrip('\n'))
        else:
            if sum > max:
                max = sum
            sum = 0
print(max)
